vllm/engine/llm_engine.py

```diff
@@ -676,7 +676,11 @@ def step(self) -> List[RequestOutput]:
         else:
             output = []
 
-        return self._process_model_outputs(output, scheduler_outputs)
+        outputs = self._process_model_outputs(output, scheduler_outputs)
```

Comment:
Will return finished outputs twice? `LLM` object will get duplicate output of same request?

Reply:
@hengxinCheung sorry, I'm not sure I understand the question. This PR doesn't change anything w.r.t. how many outputs are returned.

Reply:
I am sorry for cofusing you. Let me provide a more detsiled description. For example, request _A_ marked as finished in the current execution, but it will be scheduled in the next step. So this request will return last generated text twice? I think AsyncEngine can handle (filter) duplicate response, but LLM not. I will carefully review your implementation again. Thanks your reply.